projects/aca-content/src/lib/components/view-profile/view-profile.component.html

```diff
@@ -49,104 +78,156 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.LAST_NAME'
         <mat-divider class="app-general-dropdown-divider" />
         <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.JOB_TITLE' | translate}}</h4>
-          <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.jobTitle}}</p>
-          <input type="text" value="" formControlName="jobTitle" *ngIf="!generalSectionButtonsToggle"
-            class="app-profile-general-dropdown-input-details app-selected">
+          @if (generalSectionButtonsToggle) {
+            <p class="app-profile-general-dropdown-details">{{personDetails?.jobTitle}}</p>
+          } @else {
+            <input type="text" value="" formControlName="jobTitle" class="app-profile-general-dropdown-input-details app-selected">
+          }
         </div>
         <mat-divider class="app-general-dropdown-divider" />
         <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.LOCATION' | translate}}</h4>
-          <p class="app-profile-general-dropdown-details"  *ngIf="generalSectionButtonsToggle">{{personDetails?.location}}</p>
-          <input type="text" value="" formControlName="location" *ngIf="!generalSectionButtonsToggle"
-            class="app-profile-general-dropdown-input-details app-selected">
+          @if (generalSectionButtonsToggle) {
+            <p class="app-profile-general-dropdown-details">{{personDetails?.location}}</p>
+          } @else {
+            <input type="text" value="" formControlName="location" class="app-profile-general-dropdown-input-details app-selected">
+          }
         </div>
         <mat-divider class="app-general-dropdown-divider" />
         <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.TELEPHONE' | translate}}</h4>
-          <p class="app-profile-general-dropdown-details" *ngIf="generalSectionButtonsToggle">{{personDetails?.telephone}}</p>
-          <input type="tel" name="Telephone" value="" formControlName="telephone" *ngIf="!generalSectionButtonsToggle"
-            class="app-profile-general-dropdown-input-details app-selected">
-          <mat-error class="app-error-message" *ngIf="profileForm.get('telephone').invalid">
-            {{ 'APP.EDIT_PROFILE.INVALID_INPUT' | translate }}
-          </mat-error>
+          @if (generalSectionButtonsToggle) {
+            <p class="app-profile-general-dropdown-details">{{personDetails?.telephone}}</p>
+          } @else {
+            <input type="tel" name="Telephone" value="" formControlName="telephone" class="app-profile-general-dropdown-input-details app-selected">
+          }
+          @if (profileForm.get('telephone').invalid) {
```

Comment:
just idea - feel free to apply if you think it's good approach - to avoid calling functions in html we can get error by referring to it like that: profileForm.errors.telephone. The same for all places here

Reply:
Fixed